src/applications/appeals/996/tests/config/prefill-transformer.unit.spec.js

```diff
@@ -41,6 +41,19 @@ describe('HLR prefill transformer', () => {
     });
   });
 
+  it('should return built out template from empty non-prefill data', () => {
+    const { pages, metadata } = noTransformData;
+    const formData = { testData: noTransformData.formData.testData };
+    const noTransformActual = prefillTransformer(pages, formData, metadata);
+    // ensure transformed data is not the same object as input data
```

Comment:
Is `noTransformActual` the initial pre-fill data before there is `input`?

Reply:
`noTransformActual` (wow, I named these variables badly) is the result of processing the prefill data from the backend into save-in-progress data we'll use within the form. And yes, it is transformed before any user input.